soil-service/app/services/humus_balance_service.rb

```diff
@@ -0,0 +1,28 @@
+# Handle humus balance calculation for crops
+class HumusBalanceService
+  CONSECUTIVE_YEAR_COEFFICIENT = 1.3
+
+  attr_reader :fields
```

Comment:
nit: Redundant attr_reader use. Since this is not actually used outside of this class this could be accessed simply `@fields`